src/store/flow/block.ts

```diff
@@ -90,6 +90,11 @@ export const mutations: MutationTree<IFlowsState> = {
     const block = findBlockOnActiveFlowWith(blockId, state as unknown as IContext)
     block.exits = reject(block.exits, (e) => e.uuid === exit.uuid)
   },
+  block_updateExitVendorMetadata(state, {blockId, exitId, vendorMetadata}: {blockId: string, exitId: string, vendorMetadata: object}) {
+    const block = findBlockOnActiveFlowWith(blockId, state as unknown as IContext)
+    const exit = findBlockExitWith(exitId, block)
+    exit.vendor_metadata = vendorMetadata
+  },
```

Comment:
This mutation is used in Voto5 to update exit conditions.

Reply:
Thanks for this, I was thinking to add the same thing